llabake/Courses-Management-PS
# set up application development environment

#### What does this PR do?
* Add project build configuration

#### Description of Task to be completed?
Install dependencies and tools
* Setup Webpack
* Setup code linting with ESLint
* Setup Babel
* Add editor configuration

#### How should this be manually tested?
* Clone the repo 
* Install dependencies using `npm install`
* Run `npm start` to start the application
* You should see the browser page open on `localhost:3000`

#### What are the relevant pivotal tracker stories?
* N/A
